tools/clang/include/clang/SPIRV/SpirvModule.h

```diff
@@ -102,6 +102,9 @@ class SpirvModule {
   // Returns false otherwise (e.g. if the capability already existed).
   bool addCapability(SpirvCapability *cap);
 
+  // Returns true if the capability is in the modudle.
+  bool hasCapability(SpirvCapability &cap);
```

Comment:
Where is this implemented?

Reply:
In SpirvModule.cpp

https://github.com/microsoft/DirectXShaderCompiler/pull/6848/files#diff-afc3a7f27324e1e5fc4f39ee02c1aca0f9a40ce18946d2bb51998402c825c971R252-R255